test/python/pulse/test_builder.py

```diff
@@ -0,0 +1,969 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test pulse builder context utilities."""
+
+from math import pi
+import unittest
+
+import numpy as np
+
+from qiskit import circuit
+from qiskit import compiler
+from qiskit import pulse
+from qiskit.pulse import builder
+from qiskit.pulse import exceptions
+from qiskit.pulse import macros
+from qiskit.pulse import transforms
```

Comment:
I still feel uncomfortable that these 38 files introduce an as-of-yet unseen import style into qiskit terra. I ran this locally using the regular style and it passed the tests, so why not follow qiskit terra style?

Reply:
This *is* qiskit terra style as there is no style. Since there is no style this style is equally valid. I prefer this style for the reasons I've listed previously. While **this** module may pass, if you reverted all modules I have added this style to it would fail. Therefore, I prefer consistency across my code in this PR.